terraform/environments/cooker/platform_providers.tf

```diff
@@ -65,3 +65,11 @@ provider "aws" {
     role_arn = "arn:aws:iam::${local.environment_management.aws_organizations_root_account_id}:role/ModernisationPlatformSSOReadOnly"
   }
 }
+
```

Comment:
What does this provider do? It's not referenced in your resources so can it be removed?

Reply:
It was required when the terraform was built. Not entirely sure of its purpose but it was needed

Reply:
I don't see how it's needed as it's not referenced in the code. Can you please either remove it or provide a technical justification?

Reply:
I removed this altogether and the plan worked. It has been removed from the platform_providers.tf and this has also been pushed.

Reply:
I did a plan (and apply which failed due to the rds issue) without the provider and it all worked OK. There were no issues. This was pushed up in the last release. 
Looks like it wasn't really needed but I had been getting errors about it earlier.

Reply:
This, after a re-run without it, was found to be not needed and has been removed.